### ISSUE TITLE
Embedding GraphiQL via markdown

### ISSUE THREAD

**Issue by FrankR85** [graphql/graphiql#1733]
Is there a documentation how we can integrate `GraphiQL` in our existing markdown-based documentation?

We`d like to have something like [this](https://github.com/graphql/graphql.github.io/blob/source/src/content/learn/Learn-Queries.md):
```
# { "graphiql": true }
{
  hero {
    name
  }
}
```
Is there a tutorial or something like that on how they did that?

**Comment by PrakharUniyal** [comment]
You mean like this?
```graphql
# { "graphiql": true }
{
  hero {
    name
  }
}
```

**Comment by FrankR85** [comment]
I guess this gives me just the syntax highlighting. What I really want is the embedded GraphiQL client, like on the [Learn GraphQL](https://graphql.org/learn/queries/) website.

**Comment by acao** [comment]
@FrankR85 possibly their source code is available? I’ll circle back because we would like this feature for graphql dot org.

**Comment by saihaj** [comment]
@FrankR85 on graphql.org [MiniGraphiql](https://github.com/graphql/graphql.github.io/blob/source/src/components/Marked/MiniGraphiQL.tsx ) component is used with the [`marked`](https://www.npmjs.com/package/marked) which is not as powerful as graphiql but good enough for the docs site

**Comment by acao** [comment]
yep! you can build an editor with our codemirror-graphql or monaco-graphql quite easily

**Comment by acao** [comment]
considering closing this as it's more of a support issue. you can try asking around on the discord possibly. would be cool to have a utility for this, but then we would have to choose one of many markdown parsers to support.

it's best for a third party implentation as a library, but it would be nice if we could have a demo for something like this in 2.0

if you'd like to keep this open, can we convert it to a discussion?
